src/Components/Nav/Nav.js

```diff
@@ -1,6 +1,6 @@
 import './Nav.css'
 import { NavLink } from "react-router-dom";
-import logo from '../../images/logo.png';
+import logo from '../images/logo.png';
```

Comment:
Same change will need to be implemented once we move the images folder, as mentioned above

Reply:
++